src/content/protocols/kodiak-v3.md

```diff
@@ -0,0 +1,230 @@
+# Overview
+
+## Chain
+
+Berachain Mainnet, an EVM-Identical Layer 1 blockchain utilizing Proof-of-Liquidity 
+
+> Chain score: Low
+
+## Upgradeability
+
+The protocol's upgradeability risk is primarily concentrated in three key areas:
+
+1. **Fee Management & Protocol Parameters**
+- All core protocol fees (35% protocol fee, island fees, pool fees) can be modified by Safe Wallet 1
+- Fee tiers can be enabled/disabled, potentially breaking pool functionality
+- Treasury addresses can be changed, allowing redirection of all protocol fees
+
+2. **Implementation Control**
+- Island implementations can be updated, potentially introducing malicious code
+- Farm implementations can be whitelisted, allowing deployment of malicious farm contracts
+- PandaPool implementations can be controlled, affecting token launch security
+- DEX factory addresses can be changed, impacting graduated token liquidity
+
+3. **Emergency Controls & User Protection**
+- Staking operations can be paused, blocking new deposits
+- Reward collection can be paused, preventing users from claiming rewards
+- Emergency withdrawals can be enabled, potentially disrupting farm economics
+- Addresses can be greylisted, blocking legitimate users from staking
+
+The protocol uses three multi-sig wallets (2/4, 3/6, and 2/3) to manage these permissions. While ownership can be renounced for each contract, this would permanently lock the protocol in its current state. The high concentration of critical permissions in these wallets represents a significant upgradeability risk, as compromised keys could lead to:
+- Loss of user funds through malicious implementations
+- Blocking of user access to their funds
+- Redirection of protocol fees
+- Disruption of core protocol functionality
+
+> Upgradeability score: High
+> 
+> Justification: The protocol has multiple functions that could directly result in the theft or loss of user funds if compromised, including the ability to deploy malicious implementations and redirect funds during token graduation. These risks, combined with the concentration of critical permissions in multi-sig wallets, warrant a High risk rating.
+
+## Autonomy
+
+The protocol's autonomy is controlled by three multi-sig wallets with low thresholds:
+
+1. Safe Wallet 1 (2/4): Controls UniswapV3Factory and FarmFactory
+2. Safe Wallet 2 (2/3): Controls KodiakIslandFactory
+3. Safe Wallet 3 (2/3): Controls PandaFactory
+
+These wallets can immediately execute all critical functions without any timelock or governance process.
+
+> Autonomy score: High
+>
+> Justification: The protocol exhibits Stage 0 centralization with three low-threshold multi-sig wallets controlling all critical functions. These permissions can directly result in the theft or loss of user funds through malicious implementations and parameter changes, with no timelock or community governance safeguards.
```

Comment:
there is some misconception around the category "Autonomy" or "Dependency"

it's about external code and functionality that the protocol requires to function, eg. a bridge or an oracle - a normal DEX has usually 0 dependencies, please verify that. Also check our uniswap v3 report https://www.defiscan.info/protocols/uniswap-v3

Reply:
@newgnart you didnt update anything based on this feedback, please pay attention to this